protons/driver.py

```diff
@@ -1287,30 +1392,48 @@ def __init__(self, system, temperature, pH, prmtop, cpin_filename, integrator, p
         self.temperature = temperature
         kT = kB * temperature  # thermal energy
         self.beta = 1.0 / kT  # inverse temperature
+        self.beta_unitless = strip_in_unit_system(self.beta)   # For more efficient calculation of the work (in multiples of KT) during NCMC
```

Comment:
I think the default unit on this would be KJ/mol (not KT), so make sure that is correct in the implementation.